### PR TITLE
proof of concept

### PR DESCRIPTION
**Pull request by hixi**
@das-g please have a look ;-)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    environment:
+      - PGHOST=postgres
+    volumes:
+      - ./examples:/examples
+  # pgamin4:
+  #   image: fenglc/pgadmin4
+  #   ports:
+  #     - "5000:5050"
+  #   links:
+  #     - "pgtap:postgres"
```
Comment (das-g) [inline docker-compose.yml]: remove or (test and) reactivate commented-out code

File: examples/angproj/01-tests/dummy-test.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+-- Format the output for nice TAP.
+\pset format unaligned
+\pset tuples_only true
+\pset pager
+
+-- Revert all changes on failure.
+\set ON_ERROR_ROLLBACK 1
+\set ON_ERROR_STOP true
+\set QUIET 1
```
Comment (das-g) [inline examples/angproj/01-tests/dummy-test.sql]: Has `QUIET 1` not already been set on line 2?

File: examples/angproj/01-tests/test_results.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/examples/01-tests/dummy-test.sql ..
```
Comment (das-g) [inline examples/angproj/01-tests/test_results.txt:1]: Should this (result?) file be checked in at all?

**Comment by hixi** [comment]
@das-g makes it easier to review, but your comments were lost ^^

File: scripts/reset_db.sql [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+SET standard_conforming_strings = on;
+
+--
+-- Drop databases
+--
+
+
+
+
+
```
Comment (das-g) [inline scripts/reset_db.sql]: Is this as-of-yet empty section going to be populated?

File: scripts/create_pg_tap_extension.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
```
Comment (das-g) [inline scripts/create_pg_tap_extension.sh:2]: :+1:

File: scripts/reset_database.sh [hunk 6]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+psql -U postgres -h postgres < /opt/bin/reset_db.sql
```
Comment (das-g) [inline scripts/reset_database.sh]: Should we `set -e` here, too?

File: run_test.sh [hunk 7]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
```
Comment (das-g) [inline run_test.sh:2]: :+1:

**Review by das-g — CHANGES_REQUESTED** [review]

File: examples/angproj/01-tests/test_results.txt (hunk 3 above)
Comment (hixi) [reply examples/angproj/01-tests/test_results.txt:1]: Yes, this is just an example usage.

File: scripts/reset_db.sql (hunk 4 above)
Comment (hixi) [reply scripts/reset_db.sql]: the reset script does not work, I'm probably better off removing it than committing something broken...

**Pull request merged**